lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -267,110 +263,93 @@ describe('GatherRunner', function() {
+  it('sets up the driver to begin emulation when all flags are undefined', async () => {
+    await GatherRunner.setupDriver(driver, {
       settings: {
         emulatedFormFactor: 'mobile',
         internalDisableDeviceScreenEmulation: false,
       },
-    }).then(_ => {
-      assert.ok(tests.calledDeviceEmulation, 'did not call device emulation');
-      assert.deepEqual(tests.calledNetworkEmulation, {
-        latency: 0, downloadThroughput: 0, uploadThroughput: 0, offline: false,
-      });
-      assert.ok(!tests.calledCpuEmulation, 'called cpu emulation');
     });
+
+    connectionStub.sendCommand.findInvocation('Emulation.setDeviceMetricsOverride');
+    expect(connectionStub.sendCommand.findInvocation('Network.emulateNetworkConditions')).toEqual({
+      latency: 0, downloadThroughput: 0, uploadThroughput: 0, offline: false,
+    });
+    expect(() =>
+      connectionStub.sendCommand.findInvocation('Emulation.setCPUThrottlingRate')).toThrow();
   });
 
   it('applies the correct emulation given a particular emulationFormFactor', async () => {
-    const emulationFn = jest.fn();
-    const driver = getMockedEmulationDriver(
-      emulationFn,
-      () => true,
-      () => true
-    );
-
     const getSettings = formFactor => ({
       emulatedFormFactor: formFactor,
       internalDisableDeviceScreenEmulation: false,
     });
 
     await GatherRunner.setupDriver(driver, {settings: getSettings('mobile')});
-    expect(emulationFn.mock.calls.slice(-1)[0][0]).toMatchObject({mobile: true});
+    expect(connectionStub.sendCommand.findInvocation('Emulation.setDeviceMetricsOverride'))
+      .toMatchObject({mobile: true});
 
+    resetDefaultMockResponses();
     await GatherRunner.setupDriver(driver, {settings: getSettings('desktop')});
-    expect(emulationFn.mock.calls.slice(-1)[0][0]).toMatchObject({mobile: false});
+    expect(connectionStub.sendCommand.findInvocation('Emulation.setDeviceMetricsOverride'))
+      .toMatchObject({mobile: false});
 
-    emulationFn.mockClear();
+    resetDefaultMockResponses();
     await GatherRunner.setupDriver(driver, {settings: getSettings('none')});
-    expect(emulationFn).not.toHaveBeenCalled();
+    expect(() =>
+      connectionStub.sendCommand.findInvocation('Emulation.setDeviceMetricsOverride')).toThrow();
   });
 
-  it('stops throttling when not devtools', () => {
-    const tests = {
-      calledDeviceEmulation: false,
-      calledNetworkEmulation: false,
-      calledCpuEmulation: false,
-    };
-    const createEmulationCheck = variable => (...args) => {
-      tests[variable] = args;
-      return true;
-    };
-    const driver = getMockedEmulationDriver(
-      createEmulationCheck('calledDeviceEmulation'),
-      createEmulationCheck('calledNetworkEmulation'),
-      createEmulationCheck('calledCpuEmulation')
-    );
-
-    return GatherRunner.setupDriver(driver, {
+  it('stops throttling when not devtools', async () => {
+    // const tests = {
```

Comment:
Is this a good test? I can't tell what it's doing.

Reply:
I feel like this test _is_ testing something different, namely that when `throttlingMethod` is set to `provided` the `tests.calledCpuEmulation` was not called, so this would need to `expect.ok(!tests.calledCpuEmulation)`? But it is not very readable...I might be off base.

Reply:
this is correct. but the test above does the same. I just made that test more explicit by setting the throttling setting.